sdk/containerregistry/test-resources-post.ps1

```diff
@@ -0,0 +1,12 @@
+param (
+    [hashtable] $DeploymentOutputs,
+    [string] $TenantId,
+    [string] $TestApplicationId,
+    [string] $TestApplicationSecret
+)
+
+Import-AzContainerRegistryImage `
+    -ResourceGroupName $DeploymentOutputs['CONTAINERREGISTRY_RESOURCE_GROUP'] `
+    -RegistryName $DeploymentOutputs['CONTAINERREGISTRY_USERNAME'] `
+    -SourceImage "library/hello-world" -SourceRegistryUri "registry.hub.docker.com" `
```

Comment:
Extreme nit: use single quotes for strings when you want to be sure nothing is evaluated within them. That said, none of these have any risk of inner evaluations - its just a good habit to get into with scripts.